rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/ReplayRSpaceTests.scala

```diff
@@ -86,43 +86,6 @@ trait ReplayRSpaceTests extends ReplayRSpaceTestsBase[String, Pattern, String, S
       }
     }
 
-  /**
-    * RSpace and RepayRSpace both have a reset functionality
-    * It has been shown that it's not obvious when it can be safely skipped
-    * (iow the store is already at desired root)
-    * This test shows that an internal mechanism 'isDirty' properly identifies such cases
-    */
-  "reset to the same checkpoint" should "work" ignore fixture {
```

Comment:
no point in this test?

Reply:
It tests the `isDirty` functionality which we agreed is no longer necessary